<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Port Scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="ipLayout">
      <item>
       <widget class="QLabel" name="ipLabel">
        <property name="text">
         <string>IP地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ipLineEdit"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="portRangeLayout">
      <item>
       <widget class="QLabel" name="portRangeLabel">
        <property name="text">
         <string>端口范围：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="portRangeLineEdit"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="scanTypeLayout">
      <item>
       <widget class="QLabel" name="scanTypeLabel">
        <property name="text">
         <string>扫描类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="scanTypeComboBox">
        <item>
         <property name="text">
          <string>ICMP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TCP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UDP</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>扫描线程数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="thread_num_label">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>50</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>开始扫描</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QProgressBar" name="progress_bar">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QListWidget" name="resultListWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
